### PR TITLE
add Hacktoberfest  label

### PR DESCRIPTION
**Pull request by cpanato**
fix: https://mattermost.atlassian.net/browse/IS-205 item 2

### CODE REVIEW

**Review by jespino — APPROVED** [review]
LGTM

File: github_jira_tools/sync_helpwanted_tickets.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ def cli(jira_username, jira_token, github_token, webhook_url, dry_run, debug):
 
     log = ""
     if len(issues) > 0:
-        log = create_github_issues(jira_username, jira_token, github_token, 'mattermost/mattermost-server', ['Help Wanted', 'Up For Grabs'], issues, dry_run)
+        log = create_github_issues(jira_username, jira_token, github_token, 'mattermost/mattermost-server', ['Help Wanted', 'Up For Grabs', 'Hacktoberfest'], issues, dry_run)
```
Comment (jasonblais) [inline github_jira_tools/sync_helpwanted_tickets.py:32]: Does this only apply to `mattermost-server` repo?

Comment (jespino) [reply github_jira_tools/sync_helpwanted_tickets.py:32]: Yes, we always publish the jira tickets in `mattermost-server` repo

**Review by jasonblais — APPROVED** [review]

**Comment by cpanato** [comment]
will deploy tomorrow

**Comment by hanzei** [comment]
@jasonblais @cpanato Maybe we want to wait with the deployment until hackfest starts. Or a least until the 30th of September.

**Pull request merged**